File: tests/data-clumps/test-cases/field-field/simple/negative/typescript/source/Patient.ts
export class Patient {
  public patientId: number;
  public isActive: boolean;
  public visitCount: number;

  constructor(patientId: number, isActive: boolean, visitCount: number) {
    this.patientId = patientId;
    this.isActive = isActive;
    this.visitCount = visitCount;
  }
}

File: tests/data-clumps/test-cases/field-field/simple/negative/java/source/Patient.java
public class Patient {
    public int patientId;
    public boolean isActive;
    public int visitCount;

    public Patient(int patientId, boolean isActive, int visitCount) {
        this.patientId = patientId;
        this.isActive = isActive;
        this.visitCount = visitCount;
    }
}
